yellowbrick/features/jointplot.py

```diff
@@ -292,15 +292,6 @@ def draw_xy(self, X, y, **kwargs):
             self.y_ax.hist(y, bins=hist_bins, color=histcolor_y,
                               orientation='horizontal', **self.xy_args)
 
-    def poof(self, **kwargs):
-        """
-        Creates the labels for the feature and target variables
-        """
-
-        self.joint_ax.set_xlabel(self.feature)
-        self.joint_ax.set_ylabel(self.target)
```

Comment:
@archaeocharlie thanks for submitting the PR - I think the tests will pass if you move lines 300 and 301 to the `finalize` method, at the top.